docs/source/posts/2021/2021-06-08-week-1-sajag.rst

```diff
@@ -0,0 +1,41 @@
+Welcome to my GSoC Blog!
+========================
+
+.. post:: June 8 2021
+   :author: Sajag Swami
+   :tags: google
+   :category: gsoc
+
+Hi all! 
+I'm Sajag Swami, a sophomore at Indian Institute of Technology, Roorkee. This summer, I will be working on adding a new functionality to **FURY** which shall enable users to 
+visualise various types of proteins via different representations like Richardson aka Ribbon diagrams and molecular surface diagrams. 
+As a part of my stretch goals, I’ll try to expand protein diagrams via other representations including:
+
+1. Stick
+2. Ball and stick
+3. Wire
+4. Pipes and Planks
+5. Sphere
+
+What did you do during the Community Bonding Period?
+----------------------------------------------------
+I had weekly meetings with my mentors and other core team members. In the first meeting I got acquainted with the team members and learnt about the organisation and its goal/vision.
+In the later meetings we discussed about various representations of proteins and how to go about implementing them in FURY.
+We discussed about various libraries which can be used to parse PDB and PDBx files.
+I made a `document <https://docs.google.com/document/d/1mSoAWyXlLNrCa3hN-hiP35Lj7rURYMk5jFnWZbZp70s>`_ for the same to list pros and cons of using each library. 
+I worked upon my `previous PR <https://github.com/fury-gl/fury/pull/404>`_ too during the community bonding period and fixed its docstring syntax.
+
+As my college ended early courtesy covid, I had extra time during which I experimented and learnt more about PDB and PDBx files - the details they contain and how to parse them. 
+A small backbone visualisation  of 1mb0 protein made on FURY by extracting coordinate data of its alpha carbons - 
```

Comment:
```suggestion
A small backbone visualisation  of 1mb0 protein made on FURY by extracting coordinate data of its alpha carbons 
```

Reply:
It's better with a hyphen as I'm displaying a graphic below it. 
![image](https://user-images.githubusercontent.com/65067354/127481455-3442a1a7-0263-47db-b337-de2300213ce7.png)

Reply:
+1 with @antrikshmisri, it does not look good, I recommend removing this hyphen too.

<img width="1082" alt="Capture d’écran 2021-07-29 à 13 38 42" src="https://user-images.githubusercontent.com/23106443/127485842-d84aae20-3365-46e2-a29c-8948c2b2f53f.png">

Reply:
Furthermore, it would be better if the image is centered.

Reply:
![image](https://user-images.githubusercontent.com/65067354/127487587-c4f26a92-b5e7-4bf9-b68c-b124f8517409.png)
done!